packages/target-electron/src/deltachat/controller.ts

```diff
@@ -128,6 +117,45 @@ export default class DeltaChatController {
               typeof event === 'object' &&
               event.kind
             ) {
+              // A stupid hack.
```

Comment:
I would remove this line. Looks "from outside" as if we do "stupid hacks" ;-)

Better name it "temporary workaround" and add the "reason" for this workaround here: a line why there can be only one consumer with the current implementation of the event emitting/handling

Reply:
> Looks "from outside" as if we do "stupid hacks"

It is what it is 🤷‍♀️.

OK, I'll replace with "workaround"